src/lib/multicall/useMulticall.ts

```diff
@@ -162,9 +169,19 @@ export function useMulticall<TConfig extends MulticallRequestConfig<any>, TResul
     },
   });
 
+  const handleMutate: KeyedMutator<TResult | undefined> = useCallback(
+    (...args) => {
+      const key = stableHash(swrFullKey);
+      mutateFlagsRef.current[key] = true;
+      return mutate(...args);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [mutate, stableHash(swrFullKey)]
```

Comment:
Let's move `stableHash(swrFullKey)` outside of this callback to avoid doing it twice

Reply:
done